04-prototypical-inheritance/01-creating-your-own-prototypical-instance.js

```diff
@@ -3,7 +3,9 @@ function Shape() {
 
 }
 
-Shape.prototygplpe.duplicate = function() {
+// Shape.prototygplpe.duplicate = function() { 
+// Fixed typo.
```

Comment:
You don't need to add a comment about the change you made in the code. You can do that in your pull request message.